grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceServerTest.java

```diff
@@ -284,17 +289,24 @@ protected void configure(ServerBuilder sb) throws Exception {
             sb.workerGroup(EventLoopGroups.newEventLoopGroup(1), true);
             sb.defaultMaxRequestLength(0);
 
-            sb.serviceUnder("/", new GrpcServiceBuilder()
-                    .setMaxInboundMessageSizeBytes(MAX_MESSAGE_SIZE)
-                    .addService(new UnitTestServiceImpl())
-                    .enableUnframedRequests(true)
-                    .supportedSerializationFormats(GrpcSerializationFormats.values())
-                    .build()
-                    .decorate(LoggingService.newDecorator())
-                    .decorate((delegate, ctx, req) -> {
-                        ctx.log().addListener(requestLogQueue::add, RequestLogAvailability.COMPLETE);
-                        return delegate.serve(ctx, req);
-                    }));
+            sb.service(
+                    new GrpcServiceBuilder()
+                            .setMaxInboundMessageSizeBytes(MAX_MESSAGE_SIZE)
+                            .addService(new UnitTestServiceImpl())
+                            .enableUnframedRequests(true)
+                            .supportedSerializationFormats(GrpcSerializationFormats.values())
+                            .build(),
+                    service -> service
+                            .decorate(LoggingService.newDecorator())
+                            .decorate((delegate, ctx, req) -> {
+                                ctx.log().addListener(requestLogQueue::add, RequestLogAvailability.COMPLETE);
+                                return delegate.serve(ctx, req);
+                            }));
+            sb.service(
+                    new GrpcServiceBuilder()
+                            .addService(ProtoReflectionService.newInstance())
```

Comment:
Do you think it will be good to have a shortcut like `enableReflection()`?

Reply:
It's probably ok, but I wonder if it might cause confusion too since it would automatically add a service (e.g., they would see it in DocService and might not know where it came from). With the current way, it means all added services use `addService`.

Reply:
OK!